about/faq.rst

```diff
@@ -225,64 +154,45 @@ devices with tiny screens (fewer than 300 pixels in width), you can use
 the export option to shrink images, and set that build to be used for
 certain screen sizes in the App Store or Google Play.
 
-I have a great idea that will make Godot better. What do you think?
+How can I extend Godot? 
+-----------------------------------------------------------------
+
+For extending Godot, like creating Godot Editor plugins or adding support for additional languages, take a look at :ref:`EditorPlugins <doc_making_plugins>` and tool scripts.
+
+Also see the official blog posts on these topics:
+
+* `A look at the GDNative architecture <https://godotengine.org/article/look-gdnative-architecture>`_
+* `GDNative is here! <https://godotengine.org/article/dlscript-here>`_
+
+You can also take a look at the GDScript implementation, the Godot modules, as well as the `unofficial Python support <https://github.com/touilleMan/godot-python>`_ for Godot. This would be a good starting point to see how another third-party library integrates with Godot. 
+
+I would like to contribute! How can I get started?
 -------------------------------------------------------------------
 
-Your idea will most certainly be ignored. Examples of stuff that is
-ignored by the developers:
+Awesome! As an open-source project, Godot thrives off of the innovation and ambition of developers like you. 
+
+The first place to get started is in the `issues <https://github.com/godotengine/godot/issues>`_. Find an issue that resonates with you, then proceed to the `How to Contribute <https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#contributing-pull-requests>`_ guide to learn how to fork, modify, and submit a Pull Request (PR) with your changes.
 
--  Let's do this because it will make Godot better
--  Let's do this in Godot because another game engine does it
--  Let's remove this because I think it's not needed
--  Let's remove clutter and bloat and make Godot look nicer
--  Let's add an alternative workflow for people who prefer it
+I have a great idea for Godot. How can I share it?
+-------------------------------------------------------------------
 
-Godot developers are always willing to talk to you and listen to your feedback
-very openly, to an extent rarely seen in open source projects, but they
-will care mostly about real issues you have while using Godot, not ideas
-solely based on personal belief. Developers are interested in (for
-example):
+It might be tempting to want to bring ideas to Godot--massive core changes, some sort of mimicry of what another game engine does, or some massive alternative workflow that you'd like build into the editor. These are great and we are thankful to have such motivated people want to contribute, but Godot's focus is and always will be the core functionality as outlined in the `Roadmap <https://github.com/godotengine/godot-roadmap/blob/master/ROADMAP.md>`_, `squashing bugs and addressing issues <https://github.com/godotengine/godot/issues>`_, and conversations between Godot community members. 
```

Comment:
> Godot--massive core changes
"Godot -- resulting in massive core changes

Also as a general note it would be nice if your changes were line wrapped instead of being very very long.

Reply:
I'm going to re-do this and line-wrap everything. That was a bad move on my part removing the `\n`'s.